packages/remix/src/client/performance.tsx

```diff
@@ -175,3 +237,23 @@ export function withSentry<P extends Record<string, unknown>, R extends React.Co
   // will break advanced type inference done by react router params
   return SentryRoot;
 }
+
+export function setGlobals({
+  useEffect,
+  useLocation,
+  useMatches,
+  startTransactionOnLocationChange,
+  customStartTransaction,
```

Comment:
I guess this is still there for compatibility with the old routing instrumentation?

Reply:
Yes, we can remove this while removing the old instrumentation completely.